tests/cloud/aws/variables.tf

```diff
@@ -0,0 +1,24 @@
+/*
+   Copyright 2018 Banco Bilbao Vizcaya Argentaria, S.A.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+// Force choose instance size.
+variable "flavour" {}
```

Comment:
Do we need defaults?

Reply:
Maybe for us is useful to have a default value but for other people who wants to perform benchmark on AWS bringing an c5.2xlarge instance by default might be an issue.
We can discuss about providing an alternative variables file.